ROMFS/px4fmu_common/init.d-posix/2507_cloudship

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+#
+# @name Cloudship
+# @type Airship
+# @class Airship
+#
+# @output MAIN1 stabilator
+# @output MAIN2 motor 1 starboard
+# @output MAIN3 motor 2 port
+# @output MAIN4 rudder motor
+
+sh /etc/init.d/rc.airship_defaults
+
+param set PWM_MAX 2000
+param set PWM_MIN 1000
+
+param set MPC_MANTHR_MIN 0.00
+param set MPC_THR_MIN 0.00
+param set MPC_THR_CURVE 1
+param set MPC_THR_HOVER 0.0
+
+param set NAV_DLL_ACT 0
+param set NAV_RCL_ACT 0
+
+param set MC_ACRO_Y_MAX 360.0
+param set MPC_MAN_Y_MAX 360.0
+
+param set LNDMC_Z_VEL_MAX 0.01
+param set LNDMC_XY_VEL_MAX 0.01
+param set LNDMC_LOW_T_THR 0.01
+param set LNDMC_ROT_MAX 1.0
+
+set MIXER cloudship
+
+# Configure this as airship.
+# set MAV_TYPE 7
```

Comment:
Why is this commented?

Reply:
MAV_TYPE 7 (Airship) currently does not work. Any suggestions to including a new MAV_TYPE?